src/js/themes/hpe.js

```diff
@@ -201,6 +201,16 @@ export const hpe = deepFreeze({
       textDecoration: 'underline',
     },
   },
+  avatar: {
+    size: {
+      large: `${baseSpacing * 4}px`, // 96px
+    },
+    text: {
+      size: {
+        large: 'xxlarge', // 34px
```

Comment:
```suggestion
        large: 'xxlarge', // 36px
```
This is 36px.

Reply:
<img width="404" alt="Screen Shot 2022-10-21 at 1 31 26 PM" src="https://user-images.githubusercontent.com/12522275/197285016-da396509-779d-483e-b480-4d99b488fa77.png">

Reply:
<img width="349" alt="Screen Shot 2022-10-21 at 1 41 13 PM" src="https://user-images.githubusercontent.com/12522275/197285076-1a3e2595-f9c3-4b93-a047-57501e8a225b.png">